src/main/java/uk/gov/ons/census/casesvc/service/BlankQuestionnaireService.java

```diff
@@ -54,8 +55,9 @@ public void handleBlankQuestionnaire(
   BiFunction<Case, EventTypeDTO, Case> unreceiptCaseAndSendToField =
       (caze, causeEventType) -> {
         Metadata metadata = null;
-        if (caze.getRefusalReceived() == null && !caze.isAddressInvalid()) {
-          // Only send to fieldwork if the case is not refused or address invalid
+
+        caze.setReceiptReceived(false);
```

Comment:
Still a bit unsure about setting receipt received false here and doing it again in `unreceiptCase()` a few lines down. Did we put a comment here to explain why?